runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsOverlay.java

```diff
@@ -127,9 +189,51 @@ else if (plugin.getRoomBlacklist().contains(room.getBoss().getName().toLowerCase
 						color = Color.RED;
 					}
 
+					String bossName = room.getBoss().getName();
+					if (config.showRecommendedItems())
+					{
+						switch (RaidRoom.Boss.fromString(bossName))
+						{
+							case GUARDIANS:
+								//guardians = true;
+								itemIds.add(11920);
+								break;
+							case MUTTADILES:
+								//muttadiles = true;
+								itemIds.add(11808);
+								break;
+							case MYSTICS:
+								//mystics = true;
+								itemIds.add(12018);
+								break;
+							case SHAMANS:
+								//shamans = true;
+								itemIds.add(10925);
+								break;
+							case TEKTON:
+								//tekton = true;
+								break;
+							case VANGUARDS:
+								//vanguards = true;
+								break;
+							case VASA:
+								//vasa = true;
+								itemIds.add(13265);
```

Comment:
A lot of these suggested items are very subjective, and what's up with all the bosses doing nothing?

Reply:
I have removed the rooms(bosses) in the switch statement that did nothing.